### PR TITLE
Changed some floats to int

### PR DESCRIPTION
**Pull request by BrianMwangi21**
I tried running the program on my PC but it wouldn't work if I did not turn the floats to ints

### CODE REVIEW

File: gameObjects.py [hunk 1]
```diff
@@ -32,10 +32,10 @@ def checkRightBoundary(self, width):
     def draw(self, screen, color):
         position = (self.x, self.y)
```
Comment (CaptainDaVinci) [inline gameObjects.py]: You could type cast x, y and store in position and pass that around instead of writing the x, y tuples again.

**Comment by CaptainDaVinci** [comment]
Are you by any chance doing arithmetic operations which are floating point values with x, y coordinate. This could be the reason for the error.

File: gameObjects.py (hunk 1 above)
Comment (BrianMwangi21) [reply gameObjects.py]: True

**Comment by BrianMwangi21** [comment]
I guess when the width and height is getting divided by 2, it turns to a floating point value

**Comment by CaptainDaVinci** [comment]
In python2 the operator '/' return floor(a / b), an integer, unless a or b is a floating value.

**Comment by BrianMwangi21** [comment]
I also tried running `python2 main.py` and it didn't work. That's why in the readme I added the option `python main.py` cause that worked

**Comment by Aniq55** [comment]
@BrianMwangi21 are you on Windows?

**Comment by BrianMwangi21** [comment]
yes

**Comment by Aniq55** [comment]
@BrianMwangi21 and which version of Python are you using?

**Comment by BrianMwangi21** [comment]
python 3.5.1

**Comment by CaptainDaVinci** [comment]
Project is in python 2

**Comment by BrianMwangi21** [comment]
oh...

**Pull request closed without merging**